cfme/infrastructure/pxe.py

```diff
@@ -587,6 +625,16 @@ def step(self):
         self.prerequisite_view.toolbar.configuration.item_select('Edit this Customization Template')
 
 
+@navigator.register(CustomizationTemplateCollection, 'Delete')
```

Comment:
**REQUIRED:** The old method went into Details, and then used the toolbar from there, this navigation destination does it from the list of all images of a certain type, but it is an ambiguous name. I would propose to remove this destination and do it the old way.

Reply:
@psav we can do multiple delete from here directly, which will save our time when we run the test case if required.

Reply:
But the page isn't a ```Delete``` page. If someone wants to delete multiple System Images isn't it possible that they would be of different image_types?

Reply:
@psav understood your point, thanks for the review :)